meson.build

```diff
@@ -0,0 +1,237 @@
+project('libass', 'c', license: 'ISC',
+        meson_version: '>= 0.55.0',
+        # cpp_std specified because meson compiler options are currently global
+        default_options: ['c_std=c99', 'cpp_std=c++11', 'buildtype=debugoptimized', 'warning_level=2'],
```

Comment:
Can you explain *(again probably)* in more detail why this needs `cpp_std=c++11`?

Reply:
I'll have to go back through logs/this issue to figure out why. It's been too long.

Reply:
> because meson compiler options are currently global

This is no longer true in meson git master, see the unreleased changelog at https://mesonbuild.com/Release-notes-for-0-63-0.html#compiler-options-can-be-set-per-subproject

Reply:
Looks like this was needed for the MacOS builds on CI for some reason, because Meson took the value specified by the top project and was trying to use it for harfbuzz? I have some discord logs from Feb 2021 when this was presumably added related to that. I can try to lift it and see if whatever was going on is still relevant, or maybe remember the details of why this was useful/necessary and not just an issue to file upstream.

Reply:
> This is no longer true in meson git master, see the unreleased changelog at https://mesonbuild.com/Release-notes-for-0-63-0.html#compiler-options-can-be-set-per-subproject

Oh, nice. I guess if that's yet-unreleased I probably have to keep this in, but I should include a note.